tests/apiv2/entities.py

```diff
@@ -1,18 +1,23 @@
 import datetime
 import unittest
 
-from fastapi.testclient import TestClient
-
 from core import database_arango
 from core.schemas import entity
+from core.schemas.user import UserSensitive
 from core.web import webapp
+from fastapi.testclient import TestClient
 
 client = TestClient(webapp.app)
```

Comment:
Something to consider to avoid code repetition: creating an `AuthenticatedTestClient` class in another module that extends `TestClient` and in the init creates the user and fetches the token, and sets headers, and importing this AuthenticatedTestClient in tests.

Reply:
https://github.com/yeti-platform/yeti/issues/961